pkg/config/validation/validation.go

```diff
@@ -1072,6 +1072,21 @@ func ValidateDatadogCollector(d *meshconfig.Tracing_Datadog) error {
 	return ValidateProxyAddress(strings.Replace(d.GetAddress(), "$(HOST_IP)", "127.0.0.1", 1))
 }
 
+// ValidateStackdriverCollector validates the configuration for sending envoy spans to Stackdriver
+func ValidateStackdriverCollector(st *meshconfig.Tracing_Stackdriver) error {
+	var errs error
+	if st.GetMaxNumberOfAnnotations() == nil {
+		errs = multierror.Append(errs, errors.New("maxNumberOfAnnotations is required"))
```

Comment:
Does this mean all those params are needed in mesh config ?
I do not have expect most people to set any of this .
And empty config should suffice.

Reply:
Removing validation in that case...